src/framework/input/element-input.js

```diff
@@ -837,21 +837,29 @@ class ElementInput {
             if (hoveredNow) this._fireEvent('selectenter', new ElementSelectEvent(event, hoveredNow, camera, inputSource));
         }
 
+        const pressed = this._selectedPressedElements[inputSource.id];
+        if (eventType === 'selectmove' && pressed) {
+            this._fireEvent('selectmove', new ElementSelectEvent(event, pressed, camera, inputSource));
+        }
```

Comment:
This works like the touchmove event where it only fires on the element that it was originally clicked on

Reply:
Question here is if I should make this more like the mouse where `selectmove` is fired on the current element that it is over unless it was clicked on an existing element to start with, in which case, it would do the above.